brains-and-beards/react-native-animated-code-input
# Adjust build library

@marras @ppeszko ready for code review 

to test it:

`yarn build`
`cd example`
fresh `yarn install`
in node modules should be library folder with lib 

- [ ] check app is working
